gnc/navigator_perception/include/navigator_vision_lib/model.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MODEL_H
```

Comment:
There's nothing really wrong with this, but since all the other c++ stuff uses `#pragma once`, I think we should just stick with it for consistency.